Sources/Swiftly/Install.swift

```diff
@@ -56,25 +56,35 @@ struct Install: SwiftlyCommand {
     ))
     var token: String?
 
+    @Flag(help: "Skip PGP verification of the installed toolchain's signature.")
+    var noVerify = false
```

Comment:
While this will add a new step to installation by default, I figured it would be best to default to best practices/security. As a compromise, if `gpg` isn't installed or the keys haven't been imported, verification will just be skipped to avoid breaking any existing users. I do wonder if we should also have a `--verify` argument that will proceed with installation if and only if verification passes. WDYT?

Reply:
I think the default should be to fail if `gpg` isn't installed, or the verification fails. At both points we should ask the user if they want to continue with the default being "no". There is already a `--no-verify` argument to avoid this. We could also assume `--y` is the equivalent to `--no-verify`

I see swiftly does not import the keys for the user. Is this something it could do?

Reply:
I'd rather get this right than worry too much about breaking existing users. This is v0.x still

Reply:
>I think the default should be to fail if gpg isn't installed, or the verification fails. At both points we should ask the user if they want to continue with the default being "no". There is already a --no-verify argument to avoid this. We could also assume --y is the equivalent to --no-verify

I agree that failing in these cases is probably better, updated to do so. That said, I'd like to avoid the interactive step if possible (see my comment above). Also, `install` doesn't currently have a `-y` flag as the command doesn't have any interactive prompts.

>I see swiftly does not import the keys for the user. Is this something it could do?

`swiftly-install.sh` will do that for users, but swiftly doesn't currently. If possible, I'd prefer to keep all non-toolchain installation tasks limited to the install script to avoid having to duplicate the logic for it / separate the concerns better. If this ends up being a pain point for users I'm definitely happy to revisit the idea, though I think it should be fine since most users will get the keys during install.